docker/Dockerfile

```diff
@@ -1,50 +1,37 @@
 FROM golang:1.12.7 AS proto_base
-
 ENV GO111MODULE=off
 RUN apt-get update && apt-get -y install curl unzip
 RUN curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.9.1/protoc-3.9.1-linux-x86_64.zip && \
-    unzip protoc-3.9.1-linux-x86_64.zip 
+    unzip protoc-3.9.1-linux-x86_64.zip
 RUN go get -u github.com/golang/protobuf/protoc-gen-go
-
-
 COPY daemon/ daemon/
 COPY Makefile .
 COPY .mk/ .mk/
-
 RUN make proto
 
 FROM golang:1.12.7 AS build_base
-
 WORKDIR /go/src/github.com/networkop/meshnet-cni
 ENV GO111MODULE=on
-
 COPY go.mod .
 COPY go.sum .
-
 RUN go mod download
 
-FROM build_base AS build
 
+FROM build_base AS build
 COPY . .
-
-COPY --from=proto_base daemon/generated daemon/generated
-RUN CGO_ENABLED=0 GOOS=linux go build -o meshnet plugin/meshnet.go 
+COPY --from=proto_base /go/daemon/generated daemon/generated
```

Comment:
This is the only real change in this commit.  
- COPY --from=proto_base daemon/generated daemon/generated
+ COPY --from=proto_base /go/daemon/generated daemon/generated

Rest of the changes are just removing the extra lines within each build stage.

Reply:
yep, LGTM. I just want to test it locally before merging. will do this in the next couple of days.

Reply:
Sure. Thanks.